requirements.txt

```diff
@@ -1,5 +1,6 @@
 black==19.3b0
 gunicorn==19.9.0
 pydocstyle==4.0.1
+pytest-cov==2.8.1
```

Comment:
Should we be making this a dev requirement if it's not described in the README and is not included in the evergreen config?